src/devices/MotorHat/MotorPinProvider.cs

```diff
@@ -0,0 +1,93 @@
+﻿using System;
```

Comment:
Header is missing. Please check the existing files and copy/paste the 2 lines header.

Reply:
Done.